backend/attachRendererFiber.js

```diff
@@ -152,6 +152,7 @@ function attachRendererFiber(hook: Hook, rid: string, renderer: ReactRenderer):
     var actualDuration = null;
     var actualStartTime = null;
     var treeBaseDuration = null;
+    var selfBaseDuration = null;
```

Comment:
perhaps remove this as it's no longer being used, but it might be useful in the future?